### ISSUE TITLE
Update fetal anomaly to fetal diagnosis

### ISSUE THREAD

**Issue by colinxfleming** [colinxfleming/dcaf_case_management#786]
Thanks for creating an issue! Please fill out this form so we can be sure to have all the information we need, and to minimize back and forth.

* **What are we trying to do?** @KatSDC is making some clutch edits to special circumstances. Because we're a production system, we're going to want to update what's already in our prod db.

* **What feature or behavior is this required for?** Completing the job in #776 

* **How should we solve this issue?** By running a quick one-liner as follows in rails console:

```ruby
Patient.where(special_circumstances: 'Fetal anomaly').each { |pt| pt.update special_circumstances: pt.special_circumstances.map { |circ| circ.gsub(/Fetal anomaly/, 'Fetal diagnosis') } }
```

In a nutshell: Find people who have a circumstance `Fetal anomaly`, sub out for `Fetal diagnosis`, resave to special circumstances.

I'll be running this as part of our scheduled deployment on 1/1.

**Comment by colinxfleming** [comment]
Ran this in prod - closing.
